pypeit/scripts/deimos_flexure.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
```

Comment:
Why is this specific to DEIMOS? In my opinion it should not be. If it can only handle certain wavelength ranges that is fine. Document that, but I don't see a reason why one could not run this on LRIS, and if that is the the case I think you need to revisit the code. 

This seems to be against the pypeit philosophy of not developing spectrograph specific codes when we don't have to.

Reply:
in fact it is generic (wasn't sure it would be).
renamed to `multidet_flexure.py`

Hasn't been tested on LRIS and is unlikely to work
out of the box.

Reply:
@profxj it should work out of the box on LRIS red. If it does not, it is tuned to DEIMOS which is not how we code in PypeIt. I agree that extending the wavelengths bluer is a task for a later PR, but I'm not convinced that this should fail on LRIS. 

The other point is I don't understand why it is a multi-det only routine. It should work on single det and multidet, so why is multi-det in the name?

Reply:
@jhennawi -- If you generate an LRIS test, I'll work to make sure it passes.

As for `multidet`, that is owing to its origin but it is true it should
work for single detectors too.  I'll change it to `pypeit_multislit_flexure`.
I don't think it will work on single slits right now.